pageserver/src/http/routes.rs

```diff
@@ -973,10 +961,11 @@ async fn tenant_list_handler(
     _cancel: CancellationToken,
 ) -> Result<Response<Body>, ApiError> {
     check_permission(&request, None)?;
+    let state = get_state(&request);
 
-    let response_data = mgr::list_tenants()
-        .instrument(info_span!("tenant_list"))
```

Comment:
Was there any good reason to attach a trace to this call?

Reply:
No, the function doesn't log anything.  Removed this while removing the spurious `async` on it.